.github/workflows/e2e.yaml

```diff
@@ -179,3 +179,49 @@ jobs:
           path: report/*.xml
           key: ${{ secrets.BUILDPULSE_ACCESS_KEY_ID }}
           secret: ${{ secrets.BUILDPULSE_SECRET_ACCESS_KEY }}
+
+  e2e-gke-tests:
```

Comment:
While we might skip this in this PR I'd recommend we consider using e2e test results in buildpulse additionally to the ones we already upload above